content/en/docs/concepts/security/index.md

```diff
@@ -630,333 +631,277 @@ runtime. When a request comes to the proxy, the authorization engine evaluates
-kind: ServiceRole
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: service-admin
-  namespace: default
+ name: httpbin
+ namespace: foo
 spec:
-  rules:
-  - services: ["*"]
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   - source:
+       namespaces: ["dev"]
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.auth.claims[iss]
+     values: ["https://accounts.google.com"]
 {{< /text >}}
 
-Here is another role: `products-viewer`, which has read, `"GET"` and `"HEAD"`,
-access to the service `products.default.svc.cluster.local` in the `default`
-namespace.
+#### Policy Target
+
+Policy scope (target) is determined by `metadata/namespace` and an optional `selector`.
+
+The `metadata/namespace` tells which namespace the policy applies. If set to the
+root namespace, the policy applies to all namespaces in a mesh. The value of
+root namespace is configurable, and the default is `istio-system`. If set to a
+normal namespace, the policy will only apply to the specified namespace.
+
+A workload `selector` can be used to further restrict where a policy applies.
+The `selector` uses pod labels to select the target workload. The workload
+selector contains a list of `{key: value}` pairs, where the `key` is the name of the label.
+If not set, the authorization policy will be applied to all workloads in the same namespace
+as the authorization policy.
+
+The following example policy `allow-read` which allows `"GET"` and `"HEAD"` access to
+the workload with label `app: products` in the `default` namespace.
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRole
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: products-viewer
+  name: allow-read
   namespace: default
 spec:
+  selector:
+    matchLabels:
+      app: products
   rules:
-  - services: ["products.default.svc.cluster.local"]
-    methods: ["GET", "HEAD"]
+  - to:
+    - operation:
+         methods: ["GET", "HEAD"]
 {{< /text >}}
 
-In addition, we support prefix matching and suffix matching for all the fields
-in a rule. For example, you can define a `tester` role with the following
-permissions in the `default` namespace:
+#### Value matching
 
-- Full access to all services with prefix `"test-*"`, for example:
-   `test-bookstore`, `test-performance`, `test-api.default.svc.cluster.local`.
-- Read (`"GET"`) access to all paths with `"*/reviews"` suffix, for example:
-   `/books/reviews`, `/events/booksale/reviews`, `/reviews` in service
-   `bookstore.default.svc.cluster.local`.
+Exact match, prefix match, suffix match, and presence match are supported for most
+of the field with a few exceptions (e.g., the `key` field under `when` section,
+the `ipBlocks` under `source` section and the `ports` field under `to` section only supports exact match).
```

Comment:
the `ipBlocks` under `source` section and the `ports` field under `to` section only support~s~ exact match).